### ISSUE TITLE
Multiple verbs having same verb number

### ISSUE THREAD

**Issue by drdhaval2785** [drdhaval2785/SanskritVerb#669]
This causes errors off and on.
I will note the problem when I see it.

**Comment by drdhaval2785** [comment]
The following is the output of function `verbdataerrors()` from function.php.
This takes care of this and #668.
```
awwa!:anAdare:aww:10:0037:u:sew:अ॑ट्टँ॑:::<-->kusi!:BAzArTaH:kuMs:10:0037:u:sew:कु॑सिँ॑:1321::1374
ila!:preraRe:il:10:0084:u:sew:इ॑लँ॑:1238:1253:1290<-->cakka!:vyaTane:cakk:10:0084:u:sew:च॑क्कँ॑:1182:1198:1238
ujJa!:utsarge:ujJ:06:0024:pa:sew:उ॑ज्झँ॑:::<-->ujJa!:utsarge:udJ:06:0024:pa:sew:उ॑ज्झँ॑:::
kaKa!:hasane:kaK:01:0128:pa:sew:क॑खँ॑:81::<-->kuN:Sabde:ku:01:0128:A:aniw:कु॒ङ्:607:633:645
kaqqa!:kArkaSye:kaqq:01:0404:pa:sew:क॑ड्डँ॑:::<-->kaqqa!:kArkaSye:kadq:01:0404:pa:sew:क॑ड्डँ॑:::
kuMsa!:saMSlezaRe SlezaRe ityanye:kuMs:04:0136:pa:sew:कुं॑सँ॑:::<-->BfSu!:aDaHpatane:BfS:04:0136:pa:sew:भृ॑शुँ॑:864:884:904
kuwa!:Cedane ityeke:kuw:10:0093:A:sew:कु॑टँ॒::1293:<-->dfpa!:sandIpane ityeke:dfp:10:0093:u:sew:दृ॑पँ॑:::
kuwwa!:CedanaBartsanayoH:kuww:10:0034:u:sew:कु॑ट्टँ॑:::<-->lI:dravIkaraRe:lI:10:0034:u:sew:ली॑:1329:1348:1382
kudf!:anftaBAzaRe ityeke:kud:10:0008:u:sew:कु॑दृँ॑:::<-->CfdI!:sandIpane:Cfd:10:0008:u:sew:छृ॑दीँ॑:1337::1390
kudf!:anftaBAzaRe ityeke:kud:10:0008:u:sew:कु॑दृँ॑:::<-->juza!:paritarkane paritarpaRa ityanye:juz:10:0008:u:sew:जु॑षँ॑:1350:1365:1403
kupa!:BAzArTaH:kup:10:0146:u:sew:कु॑पँ॑:1321::1374<-->gF:vijYAne:gF:10:0146:A:sew:गॄ॑::1298:1331
kupa!:BAzArTaH:kup:10:0146:u:sew:कु॑पँ॑:1321::1374<-->SuWa!:Alasye:SuW:10:0146:u:sew:शु॑ठँ॑:1225:1240:1277
kfpa!:krapa!m kapa!m kfpAyAM gatO ca:kfp:01:0866:A:sew:कृ॑पँ॒:::<-->kfpU!:sAmarTye:kxp:01:0866:A:sew:कृ॑पूँ॒:465:487:493
kzala!:SOcakarmaRi:kzal:10:0086:u:sew:क्ष॑लँ॑:1183:1199:1239<-->bila!:Bedane:bil:10:0086:u:sew:बि॑लँ॑:1191:1207:1247
guja!:Sabde:guj:06:0096:pa:sew:गु॑जँ॑:979:1000:1022<-->tila!:snehane snehe:til:06:0096:pa:sew:ति॑लँ॑:965:987:1008
guqa!:rakzAyAm:guq:06:0097:pa:sew:गु॑डँ॑:980:1001,1013:1023<-->cala!:vilasane:cal:06:0097:pa:sew:च॑लँ॑:967:989:1010
gUra!:udyamane:gUr:10:0049:A:sew:गू॑रँ॒:1268::1321<-->granTa!:banDane:granT:10:0049:u:sew:ग्र॑न्थँ॑:1342,1353:1368:1395,1406
gUra!:udyamane:gUr:10:0049:A:sew:गू॑रँ॒:1268::1321<-->luji!:BAzArTaH:luYj:10:0049:u:sew:लु॑जिँ॑:1321:1174:1374
gF:nigaraRe:gF:06:0033:pa:sew:गॄ॑:1012:1033:1064<-->tumpa!:hiMsAyAm:tump:06:0033:pa:sew:तु॑म्पँ॑:::
granTa!:banDane:granT:10:0049:u:sew:ग्र॑न्थँ॑:1342,1353:1368:1395,1406<-->luji!:BAzArTaH:luYj:10:0049:u:sew:लु॑जिँ॑:1321:1174:1374
Gf:prasravaRe srAvaRa ityeke:Gf:10:0015:u:sew:घृ॑:1229:1244:1281<-->jala!:apavAraRe:jal:10:0015:u:sew:ज॑लँ॑:1141:1157:1195
carca!:aDyayane:carc:10:0019:u:sew:च॑र्चँ॑:1285:1303:1336<-->SraTa:prayatne prasTAna ityeke:SraTa:10:0019:u:sew:श्र॑थ॑:1380:1397:1434
cala!:BftO:cal:10:0083:u:sew:च॑लँ॑:1193:1209:<-->Dakka!:nASane:Dakk:10:0083:u:sew:ध॑क्कँ॑:1181:1197:1237
cala!:BftO:cal:10:0083:u:sew:च॑लँ॑:1193:1209:<-->Sliza!:SlezaRe:Sliz:10:0083:u:sew:श्लि॑षँ॑:1165:1180:1221
ciY:cayane:ci:10:0005:u:sew:चि॑ञ्:1212:1226:<-->sPuwi!:ityapi parihAse:sPuRw:10:0005:u:sew:स्फु॑टिँ॑:::
cukka!:vyaTane:cukk:10:0085:u:sew:चु॑क्कँ॑:1182:1198:1238<-->vila!:kzepe:vil:10:0085:u:sew:वि॑लँ॑:1190::
cuwa!:Cedane:cuw:06:0103:pa:sew:चु॑टँ॑:987:1008:1030<-->tuwa!:kalahakarmaRi:tuw:06:0103:pa:sew:तु॑टँ॑:986:1007:1029
cuqqa!:BAvakaraRe:cuqq:01:0402:pa:sew:चु॑ड्डँ॑:::<-->cuqqa!:BAvakaraRe:cudq:01:0402:pa:sew:चु॑ड्डँ॑:::
cura!:steye:cur:10:0001:u:sew:चु॑रँ॑:1132:1148:1186<-->tanu!:SradDopakaraRayoH upasargAcca dErGye:tan:10:0001:u:sew:त॑नुँ॑:1355:1370:1408
cura!:steye:cur:10:0001:u:sew:चु॑रँ॑:1132:1148:1186<-->divu!:parikUjane:div:10:0001:A:sew:दि॑वुँ॒:1279:1297,1314:1330,1347
cUrRa!:preraRe:cUrR:10:0143:u:sew:चू॑र्णँ॑:1149,1222:1165:1202,1274<-->ruza!:roze:ruz:10:0143:u:sew:रु॑षँ॑:1247::1300
CfdI!:sandIpane:Cfd:10:0008:u:sew:छृ॑दीँ॑:1337::1390<-->juza!:paritarkane paritarpaRa ityanye:juz:10:0008:u:sew:जु॑षँ॑:1350:1365:1403
jartsa!:pariBAzaRahiMsAtarjanezu:jarj:01:0813:pa:sew:ज॑र्त्सँ॑:::<-->jartsa!:pariBAzaRahiMsAtarjanezu:jarts:01:0813:pa:sew:ज॑र्त्सँ॑:::
juqa!:gatO:juq:06:0148:pa:sew:जु॑डँ॑:941,988:1009:984,1031<-->DfN:avasTAne:Df:06:0148:A:aniw:धृ॒ङ्:1014::1066
jYA:niyoge:jYA:10:0043:u:sew:ज्ञा॑:1302:1322:1355<-->SvaWi!:asaMskAragatyoH ityeke:SvaRW:10:0043:u:sew:श्व॑ठिँ॑:::
Jartsa!:pariBAzaRahiMsAtarjanezu:JarJ:01:0815:pa:sew:झ॑र्त्सँ॑:::<-->Jartsa!:pariBAzaRahiMsAtarjanezu:Jarts:01:0815:pa:sew:झ॑र्त्सँ॑:::
qipa!:saNGAte:qip:10:0098:A:sew:डि॑पँ॒:1248,1253:1267:1301,1306<-->pAla!:rakzaRe:pAl:10:0098:u:sew:पा॑लँ॑:1194::
qipa!:saNGAte:qip:10:0098:A:sew:डि॑पँ॒:1248,1253:1267:1301,1306<-->Sama!:Alocane Samo darSane na mit 1952:Sam:10:0098:A:sew:श॑मँ॒:1269:1288:1322
tanu!:SradDopakaraRayoH upasargAcca dErGye:tan:10:0001:u:sew:त॑नुँ॑:1355:1370:1408<-->divu!:parikUjane:div:10:0001:A:sew:दि॑वुँ॒:1279:1297,1314:1330,1347
tasu!:upakzaye:tas:04:0109:pa:sew:त॑सुँ॑:853:872:892<-->luwa!:viloqane:luw:04:0109:pa:sew:लु॑टँ॑:862:882:902
tuja!:hiMsAbalAdAnaniketanezu:tuj:10:0044:u:sew:तु॑जँ॑:::<-->banDa!:saMyamane iti cAndrAH:banD:10:0044:u:sew:ब॑न्धँ॑:::
tfpa!:sandIpane ityeke:tfp:10:0028:u:sew:तृ॑पँ॑:1336::1389<-->pfTa!:prakzepe:pfT:10:0028:u:sew:पृ॑थँ॑:1151::1204
tfpa!:sandIpane ityeke:tfp:10:0028:u:sew:तृ॑पँ॑:1336::1389<-->rusi!:BAzArTaH ca:ruMs:10:0028:u:sew:रु॑सिँ॑:1322::
dasa!:darSanadaMSanayoH ityapyeke:das:10:0110:A:sew:द॑सँ॒:::<-->Sulka!:atisarjane atisparSane:Sulk:10:0110:u:sew:शु॑ल्कँ॑::1217:
dfpa!:harzamohanayoH:dfp:04:0036:pa:aniw:दृ॒पँ॑:837:856:876<-->pIN:pAne:pI:04:0036:A:aniw:पी॒ङ्:784:806:823
dfpa!:harzamohanayoH:dfp:04:0036:pa:aniw:दृ॒पँ॑:837:856:876<-->lIN:SlezaRe:lI:04:0036:A:aniw:ली॒ङ्:782:804:821
Dakka!:nASane:Dakk:10:0083:u:sew:ध॑क्कँ॑:1181:1197:1237<-->Sliza!:SlezaRe:Sliz:10:0083:u:sew:श्लि॑षँ॑:1165:1180:1221
DUpa!:Dupa!' BAzArTaH:Dup:10:0303:u:sew:धू॑पँ॑:1321::1374<-->DUpa!:BAzArTaH:DUp:10:0303:u:sew:धू॑पँ॑:1321::1374
DUY:kampane:DU:10:0010:u:sew:धू॑ञ्:1351:1366:1404<-->laja:apavAraRe ityeke:laja:10:0010:u:sew:ल॑ज॑:1422::1477
DUY:kampane:DU:10:0010:u:sew:धू॑ञ्:1351:1366:1404<-->laqa!:upasevAyAm jihvonmaTane laqiH mit 1926:laq:10:0010:u:sew:ल॑डँ॑:1138:1154:1192
pakza!:parigrahe:pakz:10:0024:u:sew:प॑क्षँ॑:1148::<-->pfca!:saMyamane:pfc:10:0024:u:sew:पृ॑चँ॑:1326::1378
paTi!:gatO:panT:10:0060:u:sew:प॑थिँ॑:1166:1181:1222<-->mfza!:titikzAyAm:mfz:10:0060:u:sew:मृ॑षँ॒॑:1364:1380:1417
pelf!:gatO:pall:01:0621:pa:sew:पे॑लृँ॑:331:347:349<-->pelf!:gatO:pel:01:0621:pa:sew:पे॑लृँ॑:331:347:349
pAla!:rakzaRe:pAl:10:0098:u:sew:पा॑लँ॑:1194::<-->Sama!:Alocane Samo darSane na mit 1952:Sam:10:0098:A:sew:श॑मँ॒:1269:1288:1322
pija!:hiMsAbalAdAnaniketanezu:pij:10:0046:u:sew:पि॑जँ॑:::<-->pUrI!:ApyAyane:pUr:10:0046:u:sew:पू॑रीँ॑:1323:1341:1375
pija!:hiMsAbalAdAnaniketanezu:pij:10:0046:u:sew:पि॑जँ॑:::<-->SranTa!:sandarBe:SranT:10:0046:u:sew:श्र॑न्थँ॑:1353:1368:1406
piSa!:nASane:piS:10:0173:u:sew:पि॑शँ॑:::<-->barha!:hiMsAyAm:barh:10:0173:u:sew:ब॑र्हँ॑:1241:1256:1293
pfcI!:samparcane samparke:pfc:02:0025:A:sew:पृ॑चीँ॒:680:702:<-->zUN:prARigarBavimocane:sU:02:0025:A:sew:षू॑ङ्:681:703:720
pfTa!:prakzepe:pfT:10:0028:u:sew:पृ॑थँ॑:1151::1204<-->rusi!:BAzArTaH ca:ruMs:10:0028:u:sew:रु॑सिँ॑:1322::
pF:pAlanapUraRayoH:pF:10:0004:u:sew:पॄ॑:1146:1162:1200<-->rica!:viyojanasamparcanayoH:ric:10:0004:u:sew:रि॑चँ॑:1333:1352:1386
pF:pAlanapUraRayoH:pF:10:0004:u:sew:पॄ॑:1146:1162:1200<-->sPuqi!:parihAse:sPuRq:10:0004:u:sew:स्फु॑डिँ॑:1135::1189
praTa!:praKyAne:praT:10:0027:u:sew:प्र॑थँ॑:1150::1203<-->yu:jugupsAyAm:yu:10:0027:A:sew:यु॑:1283:1301:1334
prIY:tarpane:prI:10:0039:u:sew:प्री॑ञ्:1352:1367:1405<-->luRwa!:steye:luRw:10:0039:u:sew:लु॑ण्टँ॑:::
prIN:prItO prIRane:prI:04:0002:A:aniw:प्री॒ङ्:787:808:826<-->zivu!:tantusantAne:siv:04:0002:pa:sew:षि॑वुँ॑:756:779:794
pluza!:dAhe:pluz:04:0009:pa:sew:प्लु॑षँ॑:762,856:784,876:801,896<-->mana!:jYAne:man:04:0009:A:aniw:म॒नँ॒:817:836:857
vaBra!:gatyarTaH:baBr:01:0638:pa:sew:व॑भ्रँ॑:341::361<-->vaBra!:gatyarTaH:vaBr:01:0638:pa:sew:व॑भ्रँ॑:341::361
barha!:BAzArTaH:barh:10:0300:u:sew:ब॑र्हँ॑:1241:1256:1293<-->varha!:hiMsAyAm ityeke:varh:10:0300:u:sew:व॑र्हँ॑:1321::1374
bAhf!:prayatne:bAh:01:0732:A:sew:बा॑हृँ॒:391:408:<-->bAhf!:prayatne:vAh:01:0732:A:sew:बा॑हृँ॒:391:408:
bfha!:vfdDO bfhi!r vfhi!r vfdDO Sabde ca ityeke:bfh:01:0836:pa:sew:बृ॑हँ॑::465:<-->bfha!:vfdDO bfhi!r vfhi!r vfdDO Sabde ca ityeke:vfh:01:0836:pa:sew:बृ॑हँ॑::465:
BfMSu!:aDaHpatane:BfMS:04:0006:pa:sew:भृं॑शुँ॑:::<-->zRasu!:nirasane:snas:04:0006:pa:sew:ष्ण॑सुँ॑:::
mAna!:stamBe:man:10:0073:A:sew:मा॑नँ॒:1358:1373:1410<-->vawi!:viBAjane:vaRw:10:0073:u:sew:व॑टि॑:::
marca!:SabdArTaH ca:marc:10:0151:u:sew:म॑र्चँ॑::1243:<-->zwupa!:zwUpa! samucCrAye:stup:10:0151:u:sew:ष्टु॑पँ॑:::
mI:gatO:mI:10:0032:u:sew:मी॑:1341:1359:1394<-->sAmba!:sambanDane ityeke:sAmb:10:0032:u:sew:सा॑म्बँ॑:::
muwa!:saYcUrRane:muw:10:0101:u:sew:मु॑टँ॑:1198::1254<-->Sulba!:mAne:Sulb:10:0101:u:sew:शु॑ल्बँ॑:1196:1212:
rica!:viyojanasamparcanayoH:ric:10:0004:u:sew:रि॑चँ॑:1333:1352:1386<-->sPuqi!:parihAse:sPuRq:10:0004:u:sew:स्फु॑डिँ॑:1135::1189
ruN:gatirozaRayoH:ru:01:0319:A:aniw:रु॒ङ्:609:635:648<-->hoqf!:anAdare:hoq:01:0319:A:sew:हो॑डृँ॒:173,226::182,238
ruja!:hiMsAyAm:ruj:10:0152:u:sew:रु॑जँ॑:1324:1342:1376<-->zwUpa!:samucCrAye ityeke:stUp:10:0152:u:sew:ष्टू॑पँ॑:::
o!lasjI!:vrIqAyAm vrIqe:lajj:06:0011:A:sew:ओँ॑ल॑स्जीँ॒:::<-->o!lasjI!:vrIqAyAm vrIqe:lasj:06:0011:A:sew:ओँ॑ल॑स्जीँ॒:::
vaca!:pariBAzaRe:vac:10:0058:u:sew:व॑चँ॑:1357:1375:1412<-->zmiN:anAdare ityeke:smi:10:0058:u:sew:ष्मि॑ङ्:::
vaska!:gatyarTaH:vazk:01:0106:A:sew:व॑स्कँ॒:73:72:74<-->vaska!:gatyarTaH:vask:01:0106:A:sew:व॑स्कँ॒:73:72:74
vidx!:lABe:vid:06:0013:u:sew:वि॑दॢँ॒॑॑:1033:1052:1085<-->vyaca!:vyAjIkaraRe:vyac:06:0013:pa:sew:व्य॑चँ॑:920:941:962
vyusa!:hasane sahane cetyeke ityeke:vyus:10:0115:u:sew:व्यु॑सँ॑:::<-->Svarta!:gatyAm:Svart:10:0115:u:sew:श्व॑र्तँ॑:1205::1260
Sragi!:gatyarTaH:SraNg:01:0161:pa:sew:श्र॑गिँ॑:84:87:88<-->Sragi!:gatyarTaH:SvaNg:01:0161:pa:sew:श्र॑गिँ॑:84:87:88
Sragi!:gatyarTaH:SraNg:01:0161:pa:sew:श्र॑गिँ॑:84:87:88<-->Sragi!:gatyarTaH:svaNg:01:0161:pa:sew:श्र॑गिँ॑:84:87:88
Sragi!:gatyarTaH:SvaNg:01:0161:pa:sew:श्र॑गिँ॑:84:87:88<-->Sragi!:gatyarTaH:svaNg:01:0161:pa:sew:श्र॑गिँ॑:84:87:88
zvaska!:gatyarTaH:zvazk:01:0105:A:sew:ष्व॑स्कँ॒:::<-->zvaska!:gatyarTaH:zvask:01:0105:A:sew:ष्व॑स्कँ॒:::
zvaska!:gatyarTaH:zvazk:01:0105:A:sew:ष्व॑स्कँ॒:::<-->zvaska!:gatyarTaH:svazk:01:0105:A:sew:ष्व॑स्कँ॒:::
satra:santAnakriyAyAm:sattra:10:0448:A:sew:स॑त्र॒:1410:1428:1466<-->satra:santAnakriyAyAm:satra:10:0448:A:sew:स॑त्र॒:1410:1428:1466
sAra:dOrbalye:sara:10:0407:u:sew:सा॑र॑:1380::1434<-->sAra:dOrbalye:sAra:10:0407:u:sew:सा॑र॑:1380::1434
hrasa!:Sabde:hfs:01:0808:pa:sew:ह्र॑सँ॑::449:456<-->hrasa!:Sabde:hras:01:0808:pa:sew:ह्र॑सँ॑::449:456
```

**Comment by drdhaval2785** [comment]
Trying to correct. May take some time. Not a very interesting job.

**Comment by drdhaval2785** [comment]
Closed via many commits on 6 May 2016.
Now ther are no conflicting entries in this list.
This also closed #668
